trollmoves/server.py

```diff
@@ -129,28 +129,28 @@ def pong(self, message):
     def push(self, message):
         """Reply to push request."""
         new_msg = self._move_files(message)
-        if new_msg is None:
+        if new_msg.type is not 'err':
```

Comment:
The default return value is `None`, so this will fail with `AttributeError`. Maybe this is safer?
```suggestion
        if new_msg and new_msg.type is not 'err':
```